src/Client.php

```diff
@@ -45,6 +45,7 @@
  * @method string workers(?string $workerName = null, ?int $start = null, ?int $last = null)
  * @method string workerJobs(string $worker, ?int $minScoreTime = null)
  * @method string subscription(string $queue, $op, $topic)
+ * @method string tags(int $offset, int $count)
```

Comment:
Please, add to see default values
```suggestion
 * @method string tags(int $offset = 0, int $count = 10000)
```